tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumExtensions.cs

```diff
@@ -70,7 +70,7 @@ public static void SaveScreenshot(this IWebDriver driver, TestContext testContex
 #endif
 
             Trace.WriteLine($"Saving picture to {failurePicturePath}");
-            ss.SaveAsFile(failurePicturePath, ScreenshotImageFormat.Png);
```

Comment:
Do we not want screenshots?

Reply:
It still saves screenshots as PNG by default. In the latest version, they just removed the overload to specify the image type.

Reply:
A bit of a breaking change on their part huh? :)